assets/package.json

```diff
@@ -8,7 +8,7 @@
     "watch": "concurrently --kill-others --prefix-colors auto npm:watch-*",
     "watch-build": "webpack --mode development --stats minimal --watch",
     "watch-tsc": "tsc --noEmit --preserveWatchOutput --watch",
-    "tsc:check": "tsc --noEmit",
+    "tsc:check": "tsc --noEmit --skipLibCheck",
```

Comment:
For whatever reason, this check always fails because of issues in `node_modules/@types/react-router-dom/index.d.ts`. `node_modules` is already part of the `exclude` and I also tried adding entries for every directory up to the file with the error and nothing works. All posts about this online suggest just adding the check but maybe I shouldn't have done so blindly.

Reply:
I searched for `TS2305` in the react-router repo and found [this](https://github.com/remix-run/react-router/issues/8155#issuecomment-947898227):

> You've still got @types/react-router and @types/react-router-dom installed. Remove those, as the types are now built-in on v6.

Basically the old separate v5 types package is interfering with the new v6 built-in types. There's no upgrade to the separate types package because the correct "upgrade" is to remove it.